R/rapporttekst-funksjoner.R

```diff
@@ -10,84 +10,90 @@
 #' @description
 #' `r lifecycle::badge("experimental")`
 #'
-#' Tar inn ein vektor med tal og gjev ut ein ein vektor
-#' med' LaTeX-kommandoar for finformatert vising av tala,
+#' Tar inn ein vektor med tal og gjev ut ein vektor
+#' med LaTeX-kode for finformatert vising av tala,
 #' for eksempel med tusenskiljeteikn
 #' og med avrunding til eit fast tal desimalar.
-#' 
+#'
 #' @param x Vektor med tala ein ønskjer å få ut på LaTeX-format.
-#' @param desimalar Kor mange desimalar skal visast etter kommaet?
-#'   
+#' @param desimalar Kor mange desimalar som skal visast etter kommaet.
```

Comment:
Kva tyder `NULL`-verdien for `desimalar`? (Det er jo til og med standardverdien!)

Reply:
Ignorerer denne kommentaren. Ser det står forklart i detaljane.